### ISSUE TITLE
Inconsistent focus indicators in website footer

### ISSUE THREAD

**Issue by erosselli** [django/djangoproject.com#2535]
The website footer has two different issues related to focus indicators:
1. The Django logo on the very left of the footer has no focus indicator
2. The "Hosting by In-kind donors" link has a focus indicator of a different color than the surrounding ones 

<img width="1438" height="154" alt="django logo focused but no visible focus indicator" src="https://github.com/user-attachments/assets/8e89d71b-0909-4003-88ab-2dd1f64c4ecd" />
<img width="1440" height="146" alt="focus indicator for 'hosting by in-kind donors' link, in a light green color" src="https://github.com/user-attachments/assets/38d506b6-19c8-4c0e-bf19-ac94aaade8e4" />
<img width="1436" height="149" alt="focus indicator for all other surrounding elements, in a white color" src="https://github.com/user-attachments/assets/dc82a393-596c-447f-a3a2-8bde89f4ae8c" />

Team Mars will work on this as part of Djangonaut Space Session 6.

**Comment by Mr-Sunglasses** [comment]
Hey, Eli could you please assign it to me, I would like to work on it.

**Comment by thearyamann** [comment]
Hi, I noticed this issue was assigned yesterday.
If it becomes available or needs additional help, I would be happy to contribute.
